docs/user-manual/en/clusters.md

```diff
@@ -161,8 +161,8 @@ i.e. `jgroups-file` and `jgroups-channel`, as discussed in details as
 following:
 
 - `jgroups-file` attribute. This is the name of JGroups configuration
-  file. It will be used to initialize JGroups channels. Make sure the
-  file is in the java resource path so that Apache ActiveMQ Artemis can load it.
+  file. It will be used to initialize JGroups channels. Place the file
+  in the `etc` directory of the broker instance so that Apache ActiveMQ Artemis can load it.
```

Comment:
I would keep the previous option too. Reading the new doc it seems we can only place the file  in the `etc` directory.